Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.py

```diff
@@ -2537,7 +2537,7 @@ def archive_audit_incident_command(client: PrismaCloudComputeClient, args: dict)
             string: A string that indicates success or failure
     """
     incident_id = args.get("incident_id") or ""
-    data = {'acknowledged': True if args.get("action") == "archive" else False}
+    data = {'acknowledged': args.get('action') == 'archive'}
```

Comment:
strange, why does it prefer single quote?

Reply:
strange but I don't mind since we'll have [this](https://jira-dc.paloaltonetworks.com/browse/CIAC-7094) soon

Reply:
@kobymeir do you want me to revert this?